requirements/base.txt

```diff
@@ -1,2 +1,9 @@
-# Base requirements are empty. All of our minimum requirements are specified in setup.py's install_requires variable!
--e .
+django-filter==0.13.0
+django-js-reverse==0.7.2
+six==1.10.0
+djangorestframework==3.3.3
+django==1.9.7
+colorlog==2.6.0
+docopt==0.6.2
+https://github.com/cherrypy/cherrypy/archive/v6.2.0.zip
+https://github.com/django-mptt/django-mptt/archive/master.zip#egg=django-mptt-0.8.4
```

Comment:
Woooooooooooooooooooo! 0.8.5 is out, no more need to reference the github link:
https://github.com/django-mptt/django-mptt/releases